### DISCUSSION TITLE
請版大再幫忙更新題庫

### DISCUSSION THREAD

**Discussion by chiahsinyeh** [sdwh/sdwh.github.io#36] [General]
感謝版大做了這個題庫系統以供練習！
可以請版大再更新最新題庫嗎？因為今天上課的時候老師查詢已經有再更新80多題了，謝謝您！

**Comment by sdwh** [comment]
你好，已經更新好囉！登入網頁使用 Ctrl + F5 重新整理，如果有看到「113 年 5月 最新版」就是最新的版本囉！

**Reply by chiahsinyeh** [reply]
Oh 太感謝您了，謝謝您百忙中抽空更新考題，嘉惠我們這群在水深火熱的考生😊
